### DISCUSSION TITLE
setting view on command line

### DISCUSSION THREAD

**Discussion by gokhalen** [GLVis/glvis#256] [General]
Is there an easy way to set the view on the command line? I'm looking for something equivalent to 

```
# Setup the GLVis scene. Executed after pressing the space bar.
{
   perspective off
   view 0 0
   viewcenter 0 0
   zoom 1.0
}
```

**Comment by v-dobrev** [comment]
There's the `-k`/`--keys` command line option which can handle some of the commands that do not need input.

For example, `./glvis -m <mesh> -g <solution> -k "jlR**"` will turn off perspective (`j`), the light (`l`), set the view from top (`R`) and zoom in two times (`**`).
